### PR TITLE
Add network_manager setup to tox ci job

### PR DESCRIPTION
**Pull request by michaelgugino**

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Please regen the XML

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Jinja2
```
Comment (michaelgugino) [inline requirements.txt]: It's helpful to have a requirements file if we have to run these scripts locally.

File: sjb/actions/generate_artifacts.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 ARTIFACT_DIR="$( pwd )/artifacts/generated"
 rm -rf "${ARTIFACT_DIR}"
 mkdir "${ARTIFACT_DIR}"
-{%- for name, action in artifacts.iteritems() %}
+{%- for name, action in artifacts.items() %}
```
Comment (michaelgugino) [inline sjb/actions/generate_artifacts.py]: python3 has no iteritems for dictionaries.  items is backwards compatible, but may be marginally slower for large dictionaries.

Comment (stevekuznetsov) [reply sjb/actions/generate_artifacts.py]: Can you split this and the resulting ordering changes into a separate PR please?

**Comment by michaelgugino** [comment]
Closing in favor of disabling the sanity_checks in tox test.

**Pull request closed without merging**